src/app/(with-main-header)/boards/create/page.tsx

```diff
@@ -32,6 +32,13 @@ export default function BoardsCreate() {
     setImages((prevImages) =>
       prevImages.filter((_, index) => index !== indexToDelete),
     );
+
+    const fileInput = document.getElementById(
+      "board-images",
+    ) as HTMLInputElement;
+    if (fileInput) {
+      fileInput.value = "";
+    }
```

Comment:
```suggestion
    const fileInput = useRef<HTMLInputElement>()
    .
    .
    .
    if(fileInput.current) {
      fileInput.current.value = "";
    }
```
react에서는 DOM으로 직접 접근하는 방식이 권장되지는 않아요!
DOM에 접근해야하는 일이 있다면, useRef를 이용해서 접근하시는 게 좋습니다~~